api/openapi-spec/swagger.json

```diff
@@ -424,11 +424,19 @@
           "type": "string"
         },
         "status": {
-          "description": "Status of the condition, one of True, False, Unknown.",
+          "description": "Status of the condition, one of True, False, Unknown.\n\nPossible enum values:\n - `False`:\n - `True`:\n - `Unknown`:",
```

Comment:
We can slightly improve the code to not print the last semicolon when there is no comment?

Reply:
I think there should be some re-work on the description generation. I will put this part into another PR. In this PR, let's focus on the feature enablement part.